mms/tests/unit_tests/test_model_service_worker.py

```diff
@@ -47,6 +47,7 @@ def model_service_worker(socket_patches):
 
 
 # noinspection PyClassHasNoInit
+@pytest.mark.skip(reason="Test needs to be redone")
```

Comment:
Tests are still skipped. It may be a good idea to enable these before merging the code.

Reply:
I'm adding those in the next PR rev